### ISSUE TITLE
Zeroheight plan

### ISSUE THREAD

**Issue by mimarz** [equinor/design-system#1158]
Contact Zeroheight regarding which price plan we go for

**Comment by BeckyBrekke** [comment]
Get pricing and decide for plan
Link to pricing overview: https://statoilsrm.sharepoint.com/:b:/r/sites/Designsystem/Shared%20Documents/General/Zeroheight/enterprise%20pricing.pdf?csf=1&web=1&e=uq8jBx

> GM IT Toolbox says there are several software we pay with credit cards - but they are dept credit cards not personal: 
> We have several software paid with company credit card. Some of the software purchased for Development Toolbox (SI-IT-SIT) is paid by Thor-Andre Aresviks company card and others are paid by Kurt Ole Myrens company card. 
> 
> I am not sure if your department have a credit card to use, but it is perhaps possible for you to use Kurt Ole Myrens card too. But you need to contact Kurt Ole to clarify this.

**Comment by BeckyBrekke** [comment]
Got info from zeroheight that we can choose which plan we want. [I created this doc to summarize the differences. ](https://statoilsrm.sharepoint.com/:w:/r/sites/Designsystem/Shared%20Documents/General/Zeroheight/Zeroheight%20plans.docx?d=w0a6112443e894199a7cada606823f629&csf=1&web=1&e=Zw7lni)

I recommend the following: 
![image](https://user-images.githubusercontent.com/34247620/111638771-f2340000-87fa-11eb-9a2c-deacd2d55951.png)

**Comment by mimarz** [comment]
I suggest we go for company plan 👍

**Comment by BeckyBrekke** [comment]
I wrote to Øyvind in Slack to ask about the credit card. He's off sick today but will update after I talk with him.

**Comment by BeckyBrekke** [comment]
Chatted with Andrea - need to make a formal request with IT toolbox so will go ahead and do that.

**Comment by BeckyBrekke** [comment]
Sent in a formal request with attachments/links to our evaluation, plan information and security info. Will make a new issue based on their response.

**Comment by BeckyBrekke** [comment]
Found out there was an official form that I needed to fill out to request this on ServiceNow so I did that just now.

**Comment by BeckyBrekke** [comment]
All tasks are completed except for how Equinor will pay for this. Waiting on confirmation that we have a credit card to use for this, if not we need to find one.
